beacon-chain/core/helpers/committee_test.go

```diff
@@ -659,8 +659,8 @@ func TestPrecomputeProposerIndices_Ok(t *testing.T) {
 	assert.DeepEqual(t, wantedProposerIndices, proposerIndices, "Did not precompute proposer indices correctly")
 }
 
-func TestUpdateProposerIndicesInCache_CouldNotGetActiveIndices(t *testing.T) {
+func TestUpdateProposerIndicesInCache_SlotOutOfBound(t *testing.T) {
```

Comment:
I had to update this test to `error = slot out of bound`

Since we reshuffled `ActiveValidatorIndices` to after `StateRootAtSlot` and nil state's slot getter just returns 0